docs/90_partners.md

```diff
@@ -1,17 +1,36 @@
 # Project partners
 
-* Walther Blom, Dell Technologies, High Education & Research
-* Jaco van Dijk, Dell Technologies, Higher Education
-* Bob Dröge, Center for Information Technology, University of Groningen
-* Robbert Eggermont, Delft University of Technology
-* Kenneth Hoste, HPC-UGent
-* Geert Jan Laanstra, Electrical Engineering, Mathematics and Computer Science (EEMCS), University of Twente, Enschede
-* Caspar van Leeuwen, SURF
-* Koen Mulderij, Delft University of Technology
-* Alan O'Cais, Juelich Supercomputing Centre
-* Thomas Röblitz, University of Oslo
-* Mark Sharpley, University of Cambridge Research Computing Services Division
-* Peter Stol, Vrije Universiteit Amsterdam
-* Marco Verdicchio, SURF
-* Bas van der Vlies, SURF
-* Henk-Jan Zilverberg, Center for Information Technology, University of Groningen
+## Delft University of Technology
+* Robbert Eggermont
+* Koen Mulderij
+
+## Dell Technologies
+* Walther Blom, High Education & Research
+* Jaco van Dijk, Higher Education
+
+## HPC-UGent
+* Kenneth Hoste
+
+## J&uuml;lich Supercomputing Centre
+* Alan O'Cais
+
+## University of Cambridge
+* Mark Sharpley, Research Computing Services Division
+
+## University of Groningen
+* Bob Dröge, Center for Information Technology
+* Henk-Jan Zilverberg, Center for Information Technology
+
+## University of Twente
+* Geert Jan Laanstra, Electrical Engineering, Mathematics and Computer Science (EEMCS)
+
+## University of Oslo
+* Thomas Röblitz
+
+## Vrije Universiteit Amsterdam
+* Peter Stol
+
+## SURF
```

Comment:
Should be `SURFsara` maybe? And https://www.surf.nl/en/expertises/compute-services as URL? @casparvl?

Reply:
We are now officially 'one SURF' after merging with SURFnet, SURFmarket & SURFbureau, so probably SURF is the most future proof :)

Reply:
OK, we will leave it at SURF.
@casparvl Would you like a link to:
https://www.surf.nl/  or
https://www.surf.nl/en/expertises/compute-services

Reply:
@mulderij https://www.surf.nl/en/expertises/compute-services would be nice, thanks :)